tests/unit/modules/test_openscap.py

```diff
@@ -211,3 +212,236 @@ def test_openscap_xccdf_eval_fail_not_implemented_action(self):
                 "returncode": None,
             },
         )
+
+    def test_new_openscap_xccdf_eval_success(self):
```

Comment:
Would you add this as a pytest instead. In the directory `tests/pytests/unit/modules/test_openscap.py`.

And if you are willing to migrate all of these tests into that path as well it would be greatly appreciated.

Reply:
Done :+1: